packages/rum/src/domain/record/record.ts

```diff
@@ -28,7 +29,7 @@ export interface RecordOptions {
 
 export interface RecordAPI {
   stop: () => void
-  takeFullSnapshot: (timestamp?: TimeStamp) => void
+  takeFullSnapshot: (timestamp?: TimeStamp, serializationContext?: SerializationContext) => void
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏FMU, the `serializationContext` argument should never be the default value when invoked outside of this file. So I would make it mandatory, as it would be less error-prone (= we wouldn't inadvertently miss it).